core/font.lua

```diff
@@ -95,10 +100,20 @@ SILE.font = {
     local key = _key(options)
     if not SILE.fontCache[key] then
       SU.debug("fonts", "Looking for "..key)
-      SILE.fontCache[key] = callback(options)
+      local font = callback(options)
+      SILE.font.postLoadHook(font)
```

Comment:
If it's a font here but a face in the next function that then does something extra to make it a font, maybe keeping the naming consistent between functions would make it easier to parse for humans.

```suggestion
      local face = callback(options)
      SILE.font.postLoadHook(face)
```